sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -243,7 +242,7 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
         |  2 = true, 2L = true, 2Y = true, true = 2, true = 2L, true = 2Y,
         |  2 = false, 2L = false, 2Y = false, false = 2, false = 2L, false = 2Y
         |FROM src LIMIT 1
-    """.stripMargin)
+      """.stripMargin)
```

Comment:
Does this change seem unrelated to the current pr

Reply:
Yes, I have made some minor modifications to the previously poorly formatted codes.